source/node_test.go

```diff
@@ -361,13 +374,21 @@ func testNodeSourceEndpoints(t *testing.T) {
 			_, err := kubernetes.CoreV1().Nodes().Create(context.Background(), node, metav1.CreateOptions{})
 			require.NoError(t, err)
 
+			if tc.exposeInternalIPv6 == nil {
+				tc.exposeInternalIPv6 = new(bool)
+				*tc.exposeInternalIPv6 = true
+			}
+
+			fmt.Printf("node: %v %v\n", tc.nodeName, *tc.exposeInternalIPv6)
```

Comment:
println?

Reply:
same as above

Reply:
Not resolved. Test for logs is missing as well